packages/app/src/components/home/HomePage.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Content, Header, Lifecycle, Page } from '@backstage/core-components';
+import Grid from '@material-ui/core/Grid';
+import { RandomJokeHomePageComponent } from '@backstage/plugin-home';
+
+export const HomePage = () => {
+  return (
+    <Page themeId="home">
+      <Header title="Home" subtitle={<Lifecycle alpha />} />
```

Comment:
Regarding the lifecycle.  On a "homepage", I might not as a user infer that what is under Alpha is only "the homepage", but I'd infer it means the *entire application* I'm visiting.

In that context, I think it's not useful to provide a default for this on the homepage.

I can see it making much more sense when adding an entire page, like say if I was releasing the cost insights plugin to my org in an alpha mode or something, and the entire cost insights was alpha, but contextually here I think as a user I'd infer that the HOMEPAGE is sort of one and the same as "the entire Backstage service"?

Reply:
Yes, that's a good catch, we'll leave it out. Fixed in 0e81cc3.